src/components/pages/auth/login.tsx

```diff
@@ -63,7 +60,7 @@ export function Login({ oauth }: { oauth?: boolean }) {
     // Currenty not in use
     if (flowId) {
       kratos
-        .getSelfServiceLoginFlow(String(flowId))
+        .getLoginFlow({ id: flowId as string })
```

Comment:
We should try to avoid `as` (type assertion), since we are trying to trick the compiler with it by not making our code type safe (just in some cases it is wanted). See v.g. [this article](https://www.bytelimes.com/why-you-should-avoid-type-assertions-in-typescript/).
The previous solution was not that better (converting into string, with String object), since it could lead to bugs. The best solution would be to make a type check.
But since we are about to get rid of this whole block, it is not that important anyway.